django/core/management/commands/runserver.py

```diff
@@ -6,6 +6,7 @@
 from datetime import datetime
 
 from django.conf import settings
+from django.contrib.staticfiles.handlers import StaticFilesHandler
```

Comment:
Isn't this going or be problem if `django.contrib.staticfiles` is not part of `INSTALLED_APPS`?

I think that changes should be made to the `staticfiles` provided command instead if any.

https://github.com/django/django/blob/main/django/contrib/staticfiles/management/commands/runserver.py

The moment `django.contrib.staticfiles` is added to `INSTALLED_APPS` calling `runserver` should resolve to the override in `staticfiles`.